content/en/blog/sign-verify-plugins-rules/index.md

```diff
@@ -0,0 +1,176 @@
+---
+
+## Falco v0.36.0 and the SSC security
+
+The latest stable v0.36.0, alongside falcoctl 0.6.1 and the 0.7.0 Helm chart brought new features and new improvements in the supply chain security of Falco artifacts. The two main artifacts are plugins and rule sets as you already know. They're shipped in the OCI specification format and distributed through the official Falcosecurity OCI repositories.
+
+Software supply chain attacks aim at injecting malicious code into software components like the above, to compromise downstream users. These type of attacks are among the primary threats in the today's thread landscape.
+
+In particular, attackers abuse trust relationships existing between the different open-source stakeholders. The increase of attacks on open source software throughout the last few years, demonstrate that attackers consider them a viable means for spreading malware.
+
+## SSC safeguards
+
+Safeguards against supply chain attacks can be classified by control type: directive, preventive, detective, corrective and recovery. But there ain't no such thing as a free lunch. Besides safeguards classification, the utility to cost ratio can also be an important factor to decide where to start from improving the supply chain security of a software, and can be pretty easy to measure it.
+
+There are cheap preventive safeguards that can be implemented in open source projects especially, where stakeholders platea can be pretty wide considering the contributions. For example branch protection rules are usually simple per-code repository configurations in providers (e.g. GitHub) and alongside pull-request based flows enforcing code review quorum, are also standard best practices nowadays. The same applies to reproducible builds, dependency pinning, build steps isolation, MFA authentication to repository providers, etc.
+
+On the other side, safeguards with an high utility value can require a considerable amount of effort to be implemented from end-to-end (producer and consumer sides).
+
+MITRE and OpenSSF increased their efforts to improve SSC security recently in general and in the open source, by providing frameworks and tools that increase the utility to cost ratio.
+MITRE for example proposes an end-to-end framework to preserve the integrity of the software supply chain. OpenSSF develops the SLSA framework that groups several security best-practices for open source projects.
+
+Moreover, the in-toto Attestation Framework defines a fixed, lightweight Statement that communicates some information about the execution of the software supply chain, such as if the source was reviewed, or if it went through a SLSA conformant build process.
+
+Supply chain security is important to be constantly increased through safeguards placed as soon as possible in the supply chain ("shift left") and needs to be maintained in a continuous manner reducing the operational burden (DevSecOps) as technologies evolve and so attacks do.
```

Comment:
WDYT @LucaGuerra?

```suggestion
Supply chain security is important to be constantly increased through safeguards placed as soon as possible in the supply chain ("shift left") and needs to be maintained in a continuous manner reducing the operational burden (DevSecOps) because technologies evolve and so attacks do.
```